Header+menu

Anasteshka opened the pull request:
Created header+menu markup and CSS styles. I haven't finished with the decorative effects yet. Will add them later.

Anasteshka committed:
added header

Anasteshka committed:
Merge branch 'main' into header

Anasteshka committed:
added styles

Anasteshka committed:
Merge branch 'main' into header

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
updated styles in css

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
added menu

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
Merge branch 'main' into header

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
adjusted css styles

Anasteshka committed:
updated html

Anasteshka committed:
updated css

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
updated styles

spalnique reviewed (changes requested):
- [ ] on hover/focus for elements with visible border border-color is set to #ecd979
- [ ] on hover/focus for social buttons background-color is set to transparent

Tomorrow morning will send you a video of what you've been discussing with Sasha

Anasteshka committed:
Merge branch 'main' of https://github.com/Zymbsis/DigITall into header

Anasteshka committed:
updated css

Anasteshka committed:
updated mob menu

Anasteshka commented:
I centered the menu nav and made it flexible. 
Updated hover/focus effects. 

Based on personal liking I made the following changes: 
I changed the transition time duration for some elements (open/close-menu-btn - 100ms, menu-nav-item - 200ms). Decided not to use 300ms because it was too slow for these elements.
Open/close-menu-btn now increases and decreases by 10% accordingly instead of 20%.

Zymbsis committed:
fix-by-team-lead

Zymbsis committed:
one-more-fix

Anasteshka committed:
updated header

Anasteshka committed:
Merge branch 'header' of https://github.com/Zymbsis/DigITall into header
